tests/test_parser.py

```diff
@@ -127,26 +127,28 @@ def test_parameter(self):
     def test_var(self):
         self.assertEqual(parse_one("SELECT @JOIN, @'foo'").sql(), "SELECT @JOIN, @'foo'")
 
-    def test_annotations(self):
+    def test_comments(self):
         expression = parse_one(
             """
-            SELECT
-                a #annotation1,
-                b as B #annotation2:testing ,
-                "test#annotation",c#annotation3, d #annotation4,
-                e #,
-                f # space
+            --comment1
+            SELECT /* this won't be used */
+                a, --comment2
```

Comment:
Yeah, multi-line comments are not generated correctly at the moment, but I think it's just a matter of changing the default comment to be /* .. */ instead of -- for this to be fixed.